pkgs/servers/search/manticoresearch/default.nix

```diff
@@ -0,0 +1,90 @@
+{ lib, stdenv, fetchFromGitHub, fetchurl
+, bison, cmake, flex
+, boost, icu, mariadb-connector-c, re2
+}:
+let
+  boost_static = boost.override { enableStatic = true; };
+  columnar = stdenv.mkDerivation rec {
+    pname = "columnar";
+    version = "c16-s5"; # see NEED_COLUMNAR_API/NEED_SECONDARY_API in Manticore's GetColumnar.cmake
+    src = fetchFromGitHub {
+      owner = "manticoresoftware";
+      repo = "columnar";
+      rev = version;
+      sha256 = "sha256-iHB82FeA0rq9eRuDzY+AT/MiaRIGETsnkNPCqKRXgq8=";
+    };
+    nativeBuildInputs = [ cmake ];
+    cmakeFlags = [ "-DAPI_ONLY=ON" ];
+    meta = {
+      description = "A column-oriented storage and secondary indexing library";
+      homepage = "https://github.com/manticoresoftware/columnar/";
+      license = lib.licenses.asl20;
+      platforms = lib.platforms.all;
+    };
+  };
+  stemmer = stdenv.mkDerivation {
```

Comment:
couldn't we package the latest version of libstemmer? it is probably already packaged but i see development continued on github

Reply:
Yeah, when I first wrote this PR I was downloading the exact versions used in manticoresearch's makefiles, but I'll try using the pre-existing libstemmer.